dev/io.openliberty.mcp.internal/src/io/openliberty/mcp/internal/McpServlet.java

```diff
@@ -106,8 +107,13 @@ protected void callRequest(McpTransport transport)
 
     }
 
-    @FFDCIgnore({ JSONRPCException.class, InvocationTargetException.class, IllegalAccessException.class, IllegalArgumentException.class })
+    // Helper method for ToolResponse Error
+    public ToolResponse toErrorResponse(Throwable t) {
+        String msg = t.getMessage() != null ? t.getMessage() : t.getClass().getSimpleName();
+        return ToolResponse.error(msg);
+    }
```

Comment:
In general, small helper methods like this should be `private` and be located after the methods that use them.